Makefile

```diff
@@ -52,8 +74,9 @@ export AWS_MAX_ATTEMPTS=360
 .PHONY: build-cpu-py-37-base build-cpu-py-38-base  build-gpu-cuda-111-base build-gpu-cuda-112-base build-gpu-cuda-113-base
 build-cpu-py-37-base:
 	docker build -f Dockerfile-base-cpu \
-		--build-arg BASE_IMAGE="ubuntu:18.04" \
+		--build-arg BASE_IMAGE="$(UBUNTU_IMAGE_TAG)" \
```

Comment:
this is the only change to existing images -- we need to bump to ubuntu:20.04 for these images